src/test/resources/api/azure/users/_get/rsId.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
validate this via object mapper (do not validate by id, just uname and password)

Reply:
why id is not important?

Reply:
put method works so, that id in request can differ from id of user, so maybe we should check that this id equals